src/PhpSessionPersistence.php

```diff
@@ -44,6 +48,29 @@ class PhpSessionPersistence implements SessionPersistenceInterface
     /** @var Cookie */
     private $cookie;
 
+    /** @var string */
+    private $cacheLimiter;
+
+    /** @var int */
+    private $cacheExpire;
+
+    /** @var array */
+    private static $supported_cache_limiters = [
+        'nocache'           => true,
+        'public'            => true,
+        'private'           => true,
+        'private_no_expire' => true,
+    ];
+
+    public const CACHE_PAST_DATE  = 'Thu, 19 Nov 1981 08:52:00 GMT';
```

Comment:
Why this date in particular, and not the unix epoch?

Reply:
I was curious about that value as well :smile:
https://github.com/php/php-src/blob/master/ext/session/session.c#L1194

Reply:
Wow, that's super bizarre! I guess we should use the value for consistency with the engine... maybe throw a comment in indicating where we get the value from, though. :)

Reply:
Yeah, I used it for consistency....but I'm still wondering where (or when) it comes from....

Reply:
Probably some PHP contributor's birthday. :grinning:

Reply:
https://stackoverflow.com/questions/8194481/why-is-expires-1981 :)